curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md

```diff
@@ -0,0 +1,418 @@
+---
+id: 6690e10ebe2181212abc9652
+title: Design a Business Card
```

Comment:
```suggestion
title: Build a Business Card
```

Reply:
Do we want to strictly use "Build a (project)"? I kinda liked "Design" here, but I'm maybe we want to just stick with "Build" for consistency.

Reply:
I was thinking about this and here are my two cents

For all HTML, JS, React /Typescript, and Backend labs it should use the Build naming convention.
For select CSS labs, where appropriate, it is fine if it is Design X naming convention